docs/modules/getting-started/pages/security.adoc

```diff
@@ -0,0 +1,190 @@
+
+=== Usage
+
+To begin using self defence checks in your app, you'll need to retrieve an instance of `SecurityService`. The `SecurityService` can be retrieved using the `MobileCore#GetService` method.
+
+[source, c#]
+----
+var SecurityService = MobileCore.GetService<SecurityService>();
+----
+
+Any subsequent `MobileCore#GetService` method will return the same instance of `SecurityService`. +
+There are two ways to run self defence checks: <<Provided Self Defence Checks>> and <<Custom Self Defence Checks>>.
+
+=== Provided Self Defence Checks
+
+The Security SDK provides a set of common self defence checks enlisted into the `SecurityChecks` class.
+
+A successful Check means that the environment the application is running in is secure and no action is required, as opposed to signalling if a certain feature is enabled.
+
+For example, the Emulator Detection Check `NOT_IN_EMULATOR` will return `true` when the app is *not* running in an emulator, since this would be the more secure condition.
+
+Many of the checks are cross platform, however some of them is platform specific (for example, NOT_ROOTED is for android, while NOT_JAILBROKEN is for iOS).
+
+==== Root Access Detection (NOT_ROOTED)
+
+IMPORTANT: This check is available for Android only
+
+To help prevent your app running in a device that has been rooted the `SecurityChecks#NOT_ROOTED` check can be used.
+A device is considered rooted if *any* of the following is true:
+* A custom key has been used to sign the kernel
+* The `su` binaries are present
+
+==== Jailbreak Detection (NOT_JAILBROKEN )
+
+IMPORTANT: This check is available for iOS only
+
+To help prevent your app running in a device that has been rooted the `SecurityChecks#NOT_JAILBROKEN` check can be used.
+A devide is considered JAILBROKEN if *any* of the following is true:
+* /Applications/Cydia.app exists
+* /Library/MobileSubstrate/MobileSubstrate.dylib exists
+* /bin/bash exists
+* /usr/sbin/sshd exists
+* /etc/apt exists
+* /private/var/lib/apt/ exists
+* Applications exists that can handle the "cydia://package/com.example.package" URL
+
+==== Developer Mode Detection (NO_DEVELOPER_MODE)
+
+IMPORTANT: This check is available for Android only
+
+To detect if Developer Mode has been enabled on the device the `SecurityChecks#NO_DEVELOPER_MODE` check can be used.
+This check uses Android's link:https://developer.xamarin.com/api/type/Android.Provider.Settings+Global/[Settings.Global] class.
+
+==== Debugger Detection (NO_DEBUGGER)
+
+To detect if the application is running in Debug Mode, `SecurityChecks#NO_DEBUGGER` function can be used.
+
+[NOTE]
+====
+The check uses different approaches on Android and iOS to detect if the app is running into an emulator
```

Comment:
This check doesn't check if the device is being run in an emulator right? Let's remove this line